golang/read_file.go

```diff
@@ -0,0 +1,29 @@
+
+// Read a file from file path (given as a command line arg),
+// print line by line with line numbers
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+func main() {
+	content, _ := ioutil.ReadFile(os.Args[1])
```

Comment:
```suggestion
import (
	"fmt"
	"os"
	"strings"
)

func main() {
	content, _ := os.ReadFile(os.Args[1])
```

`ioutil.ReadFile` is deprecated